<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Planner Pink</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #FFF0F5;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="sidebarFrame">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color: white;
	border-right: 1px solid #E0E0E0;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>40</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="label_logo">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #D81B60;</string>
         </property>
         <property name="text">
          <string>üå∏ MyPlanner</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_home">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #FCE4EC;
color: #D81B60;
border-radius: 10px;
text-align: left;
padding-left: 15px;
border: none;</string>
         </property>
         <property name="text">
          <string>üè†  Inicio</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_board">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: transparent;
	color: #666;
	border-radius: 10px;
	text-align: left;
	padding-left: 15px;
}
QPushButton:hover {
	background-color: #F5F5F5;
	color: #D81B60;
}</string>
         </property>
         <property name="text">
          <string>üìã  Tablero Kanban</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_calendar">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: transparent;
	color: #666;
	border-radius: 10px;
	text-align: left;
	padding-left: 15px;
}
QPushButton:hover {
	background-color: #F5F5F5;
	color: #D81B60;
}</string>
         </property>
         <property name="text">
          <string>üìÖ  Calendario</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_settings">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: transparent;
	color: #666;
	border-radius: 10px;
	text-align: left;
	padding-left: 15px;
}
QPushButton:hover {
	background-color: #F5F5F5;
	color: #D81B60;
}</string>
         </property>
         <property name="text">
          <string>‚öôÔ∏è  Configuraci√≥n</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="mainContent" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>40</number>
       </property>
       <property name="topMargin">
        <number>40</number>
       </property>
       <property name="rightMargin">
        <number>40</number>
       </property>
       <property name="bottomMargin">
        <number>40</number>
       </property>
       <item>
        <widget class="QLabel" name="label_welcome">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>24</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #333;</string>
         </property>
         <property name="text">
          <string>Hola, &lt;b&gt;Usuario&lt;/b&gt; üëã</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_subtitle">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #666;</string>
         </property>
         <property name="text">
          <string>Estas son tus tareas pendientes para hoy:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidgetTasks">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget {
	background-color: white;
	border: 1px solid #FCE4EC;
	border-radius: 15px;
	padding: 10px;
	outline: none;
}
QListWidget::item {
	padding: 15px;
	border-bottom: 1px solid #F0F0F0;
	margin-bottom: 5px;
	color: #333;
}
QListWidget::item:selected {
	background-color: #FCE4EC;
	color: #D81B60;
	border-radius: 10px;
	border: none;
}</string>
         </property>
         <item>
          <property name="text">
           <string>üìå  Comprar materiales de oficina - 10:00 AM</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>üé®  Dise√±ar mockup de la App - 12:30 PM</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>üìû  Llamar al cliente (Reuni√≥n mensual) - 16:00 PM</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>üßò‚Äç‚ôÄÔ∏è  Clase de Yoga - 19:00 PM</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="bottomBar" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnAdd">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #D81B60;
	color: white;
	border-radius: 20px;
}
QPushButton:hover {
	background-color: #C2185B;
}</string>
            </property>
            <property name="text">
             <string>+ Nueva Tarea</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>